migration/iD10/postgresql/202303020910_IDEMPIERE-5595.sql

```diff
@@ -105,3 +105,73 @@ UPDATE AD_Field SET IsDisplayed='Y', SeqNo=330, XPosition=5,Updated=TO_TIMESTAMP
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=180,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=2907
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=190,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=2901
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=200,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=2903
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=210,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=200097
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=220,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=2904
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=230,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=2544
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=240,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=205339
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=250,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=2902
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=260,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=4398
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=270,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=2906
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=280,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=2905
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=290,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=205193
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=300,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=56333
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=310,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=56334
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=320,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=204372
+;
+
+-- Mar 2, 2023, 4:16:34 PM CET
+UPDATE AD_Field SET SeqNo=330,Updated=TO_TIMESTAMP('2023-03-02 16:16:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=10 WHERE AD_Field_ID=206410
+;
+
+UPDATE AD_Process_Para SET IsShowNegateButton='Y' WHERE AD_Process_ID IN 
```

Comment:
@nmicoud - you forgot to add this one to the oracle script too

Reply:
hi @CarlosRuiz-globalqss ,
I was just wondering why the file was not saved in notepadd++ :)
Thanks,